doc/api/deprecations.md

```diff
@@ -2860,7 +2860,7 @@ The [`crypto.Certificate()` constructor][] is deprecated. Use
 [`response.end()`]: http.html#http_response_end_data_encoding_callback
 [`response.finished`]: #http_response_finished
 [`response.writableFinished`]: #http_response_writablefinished
-[`response.writableEnded`]: #http_response_writableended
+[`response.writableEnded`]: http.html#http_response_writableended
```

Comment:
Do the two lines above also need fixing?

Reply:
@richardlau  There are 2 other PRs which are doing the same.

Reply:
@Trott isn't it better to use a single PR and commit?

Reply:
For easing backport-ability, I often try to keep doc changes isolated in separate commits. (Then if a commit doesn't apply cleanly, it's either trivial to fix or else shouldn't be backported at all.) But yeah, these probably at least should have gone in a single PR. I'll try to batch things up a bit better in the future.